x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -79,11 +79,11 @@ export class UptimePlugin
 
     if (plugins.observability) {
       plugins.observability.dashboard.register({
-        appName: 'uptime',
+        appName: 'synthetics',
         hasData: async () => {
           const dataHelper = await getUptimeDataHelper();
           const status = await dataHelper.indexStatus();
-          return status.docCount > 0;
+          return { hasData: status.docCount > 0, indices: status.indices };
```

Comment:
Nit: This function could be renamed to indicate that more than just a boolean is returned with this implementation